3-platform/shopping-cart/boot/src/test/java/com/_6core/platform/shopping/cart/boot/SCStarterTest.java

```diff
@@ -0,0 +1,10 @@
+package com._6core.platform.shopping.cart.boot;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class SCStarterTest {
```

Comment:
I think `ContextLoadTest` is better name for this test
```suggestion
class SCStarterTest {
```

Reply:
I create this from boot starter classes,
@sokimaaa , Is it should be renamed for all?